src/main/java/net/dv8tion/jda/core/events/guild/voice/GuildVoiceJoinEvent.java

```diff
@@ -20,6 +20,11 @@
 import net.dv8tion.jda.core.entities.Member;
 import net.dv8tion.jda.core.entities.VoiceChannel;
 
+/**
+ * <b><u>GuildVoiceJoinEvent</u></b>
+ * <p>Fired when a {@link net.dv8tion.jda.core.entities.Member} connects to a {@link net.dv8tion.jda.core.entities.VoiceChannel}
+ * <br>Not fired when moved from a different {@link net.dv8tion.jda.core.entities.VoiceChannel} instead. For that a {@link net.dv8tion.jda.core.events.guild.voice.GuildVoiceMoveEvent} is fired instead</br>
```

Comment:
Pretty sure </br> is not valid (and also not needed). Also: those last 2 sentences sound weird (would delete the "instead" of first one)

Reply:
Not sure what [blank] is

Reply:
oh... the closing br i meant :)